src/js/pages/Escrow/components/CardEscrowBuyer.jsx

```diff
@@ -52,37 +48,54 @@ const Funding = () => (
   </React.Fragment>
 );
 
-const PreFund = () => (
+const Funded = ({payAction}) => (
   <React.Fragment>
     <span className="bg-dark text-white p-3 rounded-circle">
-      <img src={one} alt="one" />
+      <img src={two} alt="two" />
     </span>
-    <p className="h2 mt-4">Waiting for the seller to fund an escrow</p>
-    <p>Notify the seller about the trade using  Status ecrypted p2p chat</p>
-    <Button color="primary" className="btn-lg mt-3" onClick={() => {}}>Open chat</Button>
+    <h2 className="mt-4">Funds are in the escrow. Send payment to seller.</h2>
+    <Button color="primary" className="btn-lg mt-3" onClick={() => { if(confirm("Are you sure you want this trade marked as paid?")) payAction(); }}>Mark as paid</Button>
```

Comment:
Like I said in the other review, it would be bettre to have our own Confirm modal

Reply:
Yeah. Building it in a separate PR!